build/root/var/lib/ci-artifacts/entrypoints.d/gpu_operator_run_libs.sh

```diff
@@ -13,7 +13,7 @@ prepare_cluster_for_gpu_operator() {
     fi
 
     if ! toolbox/nfd/has_gpu_nodes.sh; then
-        toolbox/cluster/scaleup.sh
+        toolbox/cluster/set_scale.sh g4dn.xlarge 1
```

Comment:
for testing & coverage purposes, I would prefer this line to be written as
```
toolbox/cluster/ensure_machineset.sh g4dn.xlarge
toolbox/cluster/set_scale.sh g4dn.xlarge 1
```
I saw that `set_scale.sh` calls `ensure_machineset` role as a dependency,
but by calling it explicitly, we ensure that `ensure_machineset.sh` script & playbook don't break without us noticing it!

Reply:
I got rid of `ensure_machineset.sh` because I felt it was a bit of an implementation detail not relevant from a toolbox user POV, so there's no longer anything that can break without us noticing.

Let me know if you disagree with that choice

Reply:
indeed, it makes sense not to have `ensure_machineset.sh`